src/app/public/modules/colorpicker/colorpicker.component.html

```diff
@@ -231,7 +231,7 @@
               'sky-btn sky-btn-link': 'modern'
             }"
             [style.backgroundColor]="color"
-            (click)="setColorFromString(color)"
+            (click)="onPresetClick(color)"
```

Comment:
Colorpicker was in dire need of some updates. To make sense of what was going on, I renamed some methods to match our current patterns. ☝️